test/test_async_repository.py

```
@@ -0,0 +1,280 @@
+        assert 0 == len(results)
+
+    @pytest.mark.asyncio
+    async def test_invalid_model_id_field(self, database):
+        class NoIdModel(BaseModel):
+            something: str
+
+        class BrokenRepository(AbstractRepository[NoIdModel]):
+            class Meta:
+                collection_name = "spams"
+
+        with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)